src/app/page.tsx

```diff
@@ -3,11 +3,15 @@ import { Footer } from '../components/Footer';
 import NewsletterForm from '../components/NewsletterForm';
 import { Map } from '../components/Map';
 import { Team } from '../components/Team';
+import Image from 'next/image';
 
 export default function Home() {
   return (
     <>
       <section className="left">
+        <div class="illustration-home">
```

Comment:
this needs to be `className` instead of `class`

Reply:
it's a React thing… I know.

Reply:
Fixed